### ISSUE TITLE
ChannelTabs no longer loads

### ISSUE THREAD

**Issue by zany130** [RobinRMC/VencordPlus#58]
### Discord Username

zany130

### What happens when the bug or crash occurs?

Channel tabs is not loading when enabled

### What is the expected behaviour?

Channel tabs loads

### How do you recreate this bug or crash?

1. enable channel tab
2. Restart discord
3. notice its not loading it is

### Errors

```
Unrecognized Content-Security-Policy directive 'prefetch-src'.
Patch by BetterBios errored (Module id is 420654): /(?<=\(\),\[(?:[A-Za-z_$][\w$]*),(?:[A-Za-z_$][\w$]*)\]=(?:[A-Za-z_$][\w$]*)\.useState\(!1\),)(.{0,200})className:((?:[A-Za-z_$][\w$]*).{0,50}.maxBioHeight\))(.{0,300})onClick:\(\)=>{.{0,300}}\)}/
 SyntaxError: Unexpected end of input
    at eval (<anonymous>)
    at D2e (patchWebpack.ts:304:31)
    at Function.set (patchWebpack.ts:80:9)
    at web.602b92fcb20f85900416.js:12:12363052
    at web.602b92fcb20f85900416.js:12:12410488
Uncaught Error: Sentry successfully disabled
    at Function.set (noTrack.ts:110:23)
    at sentry.d49f96ce9388b1370982.js:453:37896
    at sentry.d49f96ce9388b1370982.js:453:38435
  Failed to load resource: net::ERR_FILE_NOT_FOUND
```

### Request Agreement

- [X] I am using Discord Stable or tried on Stable and this bug happens there as well
- [X] I have read the requirements for opening an issue above

**Comment by RobinRMC** [comment]
The issue should be resolved with commit b3c3569ecaf5574d31cf39b125e4a5cf549f290f.

**Comment by RobinRMC** [comment]
I have confirmed that ChannelTabs loads correctly.
